dm/openapi/spec/dm.yaml

```diff
@@ -1752,6 +1752,7 @@ components:
       enum:
         - Stopped
         - Running
+        - Paused
```

Comment:
why we need Paused stage?   [This](https://github.com/pingcap/tiflow/blob/master/dm/docs/RFCS/20220124_enhance_task_manageability.md) was originally designed to remove the overlap between paused and stopped states

Reply:
ok, i will remove `Paused`, but i think i will change the status to be `Stopped` after `task stop` in other pr.